apps/web-portal/lib/middlewares.js

```diff
@@ -0,0 +1,24 @@
+import cors from 'cors'
+import bodyParser from 'body-parser'
+
+export default {
+  json: bodyParser.json({
+    limit: 1024 * 1024,
+    verify: (req, res, buf) => {
+      try {
+        JSON.parse(buf)
+      } catch (e) {
+        res.send({
+          status: 0,
+          error: {
+            code: 'BROKEN_JSON',
+            message: 'Please, verify your json',
+          },
+        })
+        throw new Error('BROKEN_JSON')
```

Comment:
Create an error `InvalidJsonError` as a sub-class from UserFacesError, and put it here, near middleware

Reply:
done